software/control/core/core.py

```diff
@@ -4433,7 +4560,6 @@ def get_surface_grid(self, x_range, y_range, num_points=50):
 
 
 class LaserAutofocusController(QObject):
```

Comment:
We should have a separate PR to update LaserAutofocusController (including make it use laserAFConfig base model)

Reply:
For this PR we can leave it as is. Also it would have been better to update LaserAutofocusController to make it use laserAFConfig before doing this PR.